age/tui.go

```diff
@@ -18,9 +18,18 @@ import (
 	"golang.org/x/term"
 )
 
+const (
+	SopsAgePasswordEnv = "SOPS_AGE_PASSWORD"
+)
+
 // readPassphrase reads a passphrase from the terminal. It does not read from a
 // non-terminal stdin, so it does not check stdinInUse.
 func readPassphrase(prompt string) ([]byte, error) {
+	password := os.Getenv(SopsAgePasswordEnv)
```

Comment:
I don't think this should be here. In fact I would remove it completely from this PR; having this as a separate PR for consideration after this PR and #1641 is merged is OK, but right now is not the time to add new features here.

Reply:
It is not easy to test without it. The previous change had this code to side load password from test.
```
reader := bufio.NewReader(os.Stdin)
pass, err = reader.ReadString('\n')
if err == io.EOF {
	err = nil
}
```
Any idea how would I supply password from tests for /dev/tty?

Reply:
Hmm, in that case, use https://stackoverflow.com/a/78532310 so that it only works in testing, and not in production builds.

Reply:
done